frontend/src/components/dashboard/settings/add-credential-modal/AddCredentialModal.tsx

```diff
@@ -146,14 +146,24 @@ const AddCredentialModal = ({
       <>
         {credInput}
         <div className="separator"></div>
-        <NextButton
-          onClick={nextFunc}
-          loadingPlaceholder={
-            <>
-              Validating<i className="bx bx-loader-alt bx-spin"></i>
-            </>
-          }
-        />
+        <div className={styles.actionButtons}>
+          <PrimaryButton
```

Comment:
So when do we use a NextButton instead of PrimaryButton?

Reply:
NextButton is mainly a convenience component with a default style (`progress-button`) and button text (Next ->). It's mostly just used in the campaign creation flow. I think apart from the campaign creation flow, we should just directly use PrimaryButton. Haven't done a very thorough comb through to ensure this is consistent though.